binary/src/Cardano/Binary/ToCBOR.hs

```diff
@@ -404,6 +409,15 @@ withWordSize x =
       | otherwise                              -> 9
 
 
+instance ToCBOR Encoding where
+  toCBOR = id
+
+instance ToCBOR (Tokens -> Tokens) where
+  toCBOR = Encoding
+
+instance ToCBOR Term where
+  toCBOR = encodeTerm
+
```

Comment:
oh fun! this looks really helpful

Reply:
It actually is, I'll show later how it can be very nicely leveraged.